Framework/PythonInterface/test/python/plugins/algorithms/SANSDarkRunBackgroundCorrectionTest.py

```diff
@@ -0,0 +1,763 @@
+﻿import unittest
```

Comment:
This test is taking too long to run *(>5 seconds)*. Could you please consider refactoring the tests to get the execution time well below 1 second. You might want to move some of the responsibility to system tests.